mpas_analysis/ocean/index_nino34.py

```diff
@@ -397,7 +397,7 @@ def _running_mean(self, inputData, wgts):  # {{{
         """
 
         nt = len(inputData)
-        sp = (len(wgts) - 1)/2
+        sp = int((len(wgts) - 1)/2)
```

Comment:
It's probably better to use integer division here instead of a type case:

```python
sp = (len(wgts) - 1) // 2

Reply:
Okay, agreed. I'll edit that commit, then.  In which case, I'm *not* done after all...